src/objects/js/components/admin/search-toggle.js

```diff
@@ -0,0 +1,20 @@
+function mountSearchHelpToggle() {
+    const toggleLink = document.getElementById("toggle-search-help");
+    const content = document.getElementById("search-help-content");
+
+    if (!toggleLink || !content) return;
+
+    toggleLink.addEventListener("click", function (e) {
+        e.preventDefault();
+
+        if (content.style.display === "none" || content.style.display === "") {
+            content.style.display = "block";
+            toggleLink.textContent = "Hide search information";
+        } else {
+            content.style.display = "none";
+            toggleLink.textContent = "Show search information";
```

Comment:
These are now untranslated, I think you'll have to define them in the view and pass them to the frontend to make them translatable